core/templates/dev/head/tests/console_errors.scripts.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Scripts needed for console_error page.
+ */
+
+require('tests/console_errors.module.ts');
```

Comment:
Ordering? I think that this line should be at the bottom.

Reply:
It should be at the top since the module needs to be initialized before other things.

Reply:
I've added a comment explaining this